src/drivers/sifive_fe310-g000_pll.c

```diff
@@ -270,6 +272,11 @@ static int find_closest_config(long ref_hz, long rate) {
     return closest_index;
 }
 
+int __metal_driver_riscv_clint0_command_request(
+    struct metal_interrupt *controller, int command, void *data);
+int __metal_driver_sifive_clic0_command_request(
+    struct metal_interrupt *controller, int command, void *data);
+
```

Comment:
These declarations should be in a header file that is included both here and by the file defining the functions so that these declarations can be typechecked against the definitions.

Reply:
Moved to the header files for the CLIC/CLINT drivers.